app/src/main/java/fraktal/io/android/demo/timer/ui/TimerViewState.kt

```diff
@@ -6,7 +6,10 @@ import fraktal.io.android.demo.timer.domain.TimerCommand
 @Stable
 data class TimerViewState(
     val timerText: String,
-    val buttons: List<ButtonState>
+    val buttons: List<ButtonState>,
```

Comment:
Oh, I'm looking now, and I realize that I didn't do a very good thing about jetpack compose. because there are subtleties in updating lists.

So.
We need to talk about this moment. I specifically divided it up domain - view state. Because view state can be carry special data types for compose or special annotations. But why are we copying data from the domain model inside the view state? is this a limitation of the dimap function?

Reply:
ohh, I see your point. Sorry, my mistake.

Maybe a better approach is to start using View from FModel, not just Decider. The thing is that you recommend two different States/S: first in Decider (TimerState) and second in your View/Presentation (TimerViewState). Both of these states are built out of the same events, just now very much independent. This is CQRS. 

This way your TimerState is used only in decider / descision making component that is publishing events. 
TimerViewState is the state of the View, and you can have many different Views for one Decider. Freeeedom! 

Let me create a new PR to reflect my idea!

_Currently, you have TimerViewState and TimerState and there is a `map` function between them. Essentially, these are different variations of the same state, and can be limited. That limitation is more obvious in FModel because in order to map the State1 to State2 you need to provide two map functions from both sides (S1 -> S2 and S2 -> S1)._